src/umpire/util/memory_sanitizers.hpp

```diff
@@ -7,6 +7,10 @@
 #ifndef UMPIRE_memory_sanitizers_HPP
 #define UMPIRE_memory_sanitizers_HPP
 
+#include "umpire/config.hpp"
+
+#if defined(UMPIRE_HAS_ASAN)
```

Comment:
What's the point of leaving all the extra logic in this file if we already know the header is available?

Reply:
Agreed.  Removed